Form cache not created in CLI mode

#### Software versions 
Briefcase v1.8.0

#### Problem description
Crash when trying to export a form using CLI:
```
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.jdt.internal.jarinjarloader.JarRsrcLoader.main(JarRsrcLoader.java:58)
Caused by: java.lang.UnsupportedOperationException: getFormFileMd5Hash
	at org.opendatakit.briefcase.util.NullFormCache.getFormFileMd5Hash(NullFormCache.java:9)
	at org.opendatakit.briefcase.util.FileSystemUtils.getBriefcaseFormList(FileSystemUtils.java:134)
	at org.opendatakit.briefcase.ui.BriefcaseCLI.run(BriefcaseCLI.java:185)
	at org.opendatakit.briefcase.ui.MainBriefcaseWindow.main(MainBriefcaseWindow.java:177)
	... 5 more
```

#### Steps to reproduce the problem
`java -jar build/libs/ODK\ Briefcase\ v1.8.0.jar --form_id all-widgets --storage_directory ./ --export_directory ~/Desktop --export_filename foo.csv`

--- a/src/org/opendatakit/briefcase/ui/BriefcaseCLI.java
+++ b/src/org/opendatakit/briefcase/ui/BriefcaseCLI.java
@@ -114,6 +114,10 @@ public class BriefcaseCLI {
             log.info("Directory found, using " + f.getAbsolutePath());
         }
 
+        if (BriefcasePreferences.appScoped().getBriefcaseDirectoryOrNull() != null) {
+            FileSystemUtils.createFormCacheInBriefcaseFolder();
+        }
+
         if (odkDir != null) {
             // importing from ODK dir
             List<FormStatus> statuses = new ArrayList<FormStatus>();

